omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -53,6 +54,7 @@
 public class UserFormController {
 	
 	protected static final Log log = LogFactory.getLog(UserFormController.class);
+	protected Boolean addToProviderTable; //Need alternative rather than declaring global
```

Comment:
@rkorytkowski I could find no workaround other than using a global variable. Is it ok/ can you suggest something else?